kalastatic.module

```diff
@@ -73,3 +73,93 @@ function kalastatic_preprocess_html(&$variables) {
     $variables['build_path'] = base_path() . 'kalastatic/';
   }
 }
+
+/**
+* Prepare the variables from a text_with_summary field.
+ */
+function kalastatic_prepare_field__text_with_summary($field, $entity) {
+  $output = [];
+  foreach ($entity->$field->getValue() as $i => $item) {
```

Comment:
Text w/summary might be one of the easier places to try using the rendered output. We may even want to consider adding a nested layer to the template data to support `text.value` and `text.summary` or something like that.